src/EA.Weee.Web/Areas/Producer/Views/ProducerSubmission/AppropriateSignatory.cshtml

```diff
@@ -18,13 +17,43 @@
     {
         @Html.AntiForgeryToken()
         @Html.Gds().ValidationSummary()
-        
+
         <header class="hgroup">
             <h1 class="govuk-heading-l">@title</h1>
         </header>
 
+        <h2 class="govuk-heading-s">Appropriate signatory</h2>
+
+        <div class="govuk-body">This submission must be made by the appropriate person for your organisation.</div>
+
+        <div class="govuk-body">
+            For companies this means a director or company secretary. For partnerships this means a partner or member of the partnership. For individuals or sole traders this means that individual. For any other type of organisation this means a person who has control or management of that body.
+            <br />
```

Comment:
@stevelee-civica Dont use BR here as screenreadrs dont like it. Break the text using multiple<p> tags